### PR TITLE
shutdown any hanging simulators before startup.

### PR DESCRIPTION
**Pull request by rafael-chavez**
opts pass in address as a capability, not host.

### CODE REVIEW

**Comment by rafael-chavez** [comment]
@imurchie @Jonahss

File: lib/driver.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ class XCUITestDriver extends BaseDriver {
       this.wda = new WebDriverAgent({
         sim: this.sim, //TODO can't be a sim if real device. find a different way to supply logs to webdriveragent
         platformVersion: this.opts.platformVersion,
-        host: this.opts.host,
+        host: this.opts.address,
```
Comment (imurchie) [inline lib/driver.js]: Why change this? This is making all the functional tests fail.

Comment (rafael-chavez) [inline lib/driver.js]: when used in conjunction with appium top level project we use 'address' in our parsed args [here](https://github.com/appium/appium/blob/master/lib/parser.js#L23). Do you think we should support both so this also works when the driver is used standalone? for example:

```
host: this.opts.host || this.opts.address,
```

Comment (imurchie) [inline lib/driver.js]: Ah! That makes sense. No, I don't think we want to proliferate the number of ways to do things. That seems like a future headache in the making.

Can you, as part of this PR, change `host` to `address` here: https://github.com/appium/appium-xcuitest-driver/blob/master/lib/server.js ? That will make it have `this.opts.address` in the driver itself!

Comment (rafael-chavez) [inline lib/driver.js]: of course updated! @imurchie

**Comment by imurchie** [comment]
Thanks for this catch! I haven't been testing through the overarching system.

**Comment by imurchie** [comment]
Thanks again! I have a few more pull requests to merge, then I will publish the new version.

**Comment by imurchie** [comment]
Published as `appium-xcuitest-driver@2.0.1`.

**Pull request merged**